sdk-tools/src/sign.rs

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+//! Builder wrapper around SgxSign.
+
+use std::{
+    path::{Path, PathBuf},
+    process::Command,
+};
+
+/// This structure wraps the execution of sgx_sign commands.
+#[derive(Clone, Debug)]
+pub struct SgxSign {
```

Comment:
:thought_balloon: `sgx` is used in many places in this crate. I know the command line utility is called `sgx_sign` but I wonder if this struct should be named `EnclaveSigner` or similar.

Reply:
I think `SgxSign` is fine, because the scope is just to wrap this tool. If we want to streamline a higher-level task, they should be their own things that call this.